Build.cmake

```diff
@@ -60,8 +61,8 @@ ELSEIF(${CMAKE_CXX_COMPILER_ID} STREQUAL "GNU")
    ENDIF()
 ELSEIF(MSVC)
    set(PLATFORM_LINK_LIBRIES dbghelp)
-   set(CMAKE_CXX_FLAGS_RELEASE "/MT")
-   set(CMAKE_CXX_FLAGS_DEBUG "/MTd")
+   # set(CMAKE_CXX_FLAGS_RELEASE "${CMAKE_CXX_FLAGS_RELEASE} /MD")
```

Comment:
Under windows there is no scenario I see where it would be desirable to link against the static Runtime. The DLL runtime is the default, so I removed these flags. As mentioned above using /MTx and building as a DLL or linking against anything that uses the dynamic runtime will result in errors from crtdbg.h in Debug builds (and god knows what in Release builds) as g3logger and the executable or DLL using it will effectively use different heaps.